addon/templates/components/table-columns.hbs

```diff
@@ -42,6 +42,7 @@
       itemClassNames=(concat 'table-row ' mergedRowClasses)
       content=table.content
       defaultHeight=rowHeight
+      didMountCollection="didRenderTable"
```

Comment:
We should be able to use `(action table.didRenderTable)` and avoid having to bubble things up